src/stories/index.stories.js

```diff
@@ -1,29 +1,49 @@
 import { action } from "@storybook/addon-actions";
 import { linkTo } from "@storybook/addon-links";
 
-import MyButton from "../components/MyButton.vue";
+import InputCpt from "../components/Input.vue";
+import TextCpt from "../components/Text.vue";
+import ButtonDefault from "../components/Button.vue";
+import SearchFilterCpt from "../components/SearchFilter.vue";
+import ResultCountCpt from "../components/ResultCount.vue";
 
 export default {
-  title: "Button",
+  title: "Atomic",
 };
 
-export const withText = () => ({
-  components: { MyButton },
-  template: '<my-button @click="action">Hello Button</my-button>',
-  methods: { action: action("clicked") },
+export const Input = () => ({
+  render() {
+    return <InputCpt />;
+  },
 });
 
-export const withJSX = () => ({
+export const Text = () => ({
   render() {
-    return (
-      <MyButton onClick={linkTo("Button", "With Some Emoji")}>
-        With JSX
-      </MyButton>
-    );
+    return <TextCpt text="Find your move" />;
   },
 });
 
-export const withSomeEmoji = () => ({
-  components: { MyButton },
-  template: "<my-button>😀 😎 👍 💯</my-button>",
+export const Button = () => ({
+  render() {
+    return <ButtonDefault>Search</ButtonDefault>;
+  },
+});
+
+const SearchFilterTemplate = (args) => ({
+  components: { SearchFilterCpt },
+  setup() {
+    return { args };
+  },
+  template: '<SearchFilterCpt v-bind="args" />',
+});
+
+export const SearchFilter = SearchFilterTemplate.bind({});
+SearchFilter.args = {
+  label: "Search by",
+};
+
+export const ResultCount = () => ({
+  render() {
+    return <ResultCountCpt count="{{7}}" />;
```

Comment:
is not working, use `<ResultCountCpt count="7" />`

Reply:
@rnesteruk13 fixed